index.js

```diff
@@ -182,6 +197,17 @@ Reverb.prototype.onGatewayCommand = function(command) {
             id: body.key.entryId
           , user: body.key.registeredUserId
         });
+    } else if (command.command == 'PUSH_NOTIFICATION_CHANGE') {
+        var body = JSON.parse(command.payload);
+        if (body.notificationId == "t1") {
```

Comment:
Would it make more sense to switch on the first character here instead of testing the id exactly? In case there's ever a `t2`, etc.